java/pom.xml

```diff
@@ -82,7 +82,7 @@
       <dependency>
         <groupId>com.google.guava</groupId>
         <artifactId>guava</artifactId>
-        <version>20.0</version>
+        <version>26.0</version>
```

Comment:
As I mentioned in the internal review, this version does not exist. The versions are`26.0-jre` and `26.0-android`. Since protobuf supports Java 7, it needs to use `26.0-android`.

Reply:
Many things that depend on protobuf are on 20.0. This is likely to introduce convergency issues downstream. I wish we could do a multirepo PR like we can in google3.

Reply:
I created https://github.com/protocolbuffers/protobuf/pull/5630 with 26.0-android instead. Please take a look.